test/kwtool_web/controllers/upload_controller_test.exs

```diff
@@ -4,7 +4,7 @@ defmodule KwtoolWeb.UploadControllerTest do
   import KwtoolWeb.Support.ConnHelper
 
   describe "get index/2" do
-    test "returns 200 status", %{conn: conn} do
+    test "returns 200 status after the upload succeeded", %{conn: conn} do
```

Comment:
Again here, minor preference to use `given` for consistency:

```suggestion
    test "returns 200 status given the upload succeeded", %{conn: conn} do
```

This way `given` works in all kinds of tests.

Reply:
Updated! Thanks 👍